Ds3/Helpers/Jobs/Job.cs

```diff
@@ -127,9 +128,12 @@ public void Transfer(Func<TItem, Stream> createStreamForTransferItem)
                     {
                         this.TransferBlob(item.Client, item.Blob);
                     }
-                    catch (Exception)
+                    catch (Exception ex)
                     {
                         this._chunkStrategy.Stop();
+
+                        //if we have an OnFailure event than invoke that with the blob name, offset and the exception
```

Comment:
Confusingly worded. Remove the second "that" for easier reading. Also, I think it should be "then" instead of "than" in this instance.